docs/framework-components/specification.md

```diff
@@ -28,7 +28,7 @@ The HTTP interface is designed to handle bulk request-response communication tha
 
 ### Sparkplug Interface
 
-* A component **MUST** publish a Sparkplug birth certificate over MQTT, which identifies the provided component and publishes the URL of the HTTP endpoint. Each birth certificate can register only one component function with the Directory component, regardless of whether the component is a Sparkplug Node or Device. If a single software process intends to provide multiple components, it can publish multiple Device birth certificates to achieve this.
+* A component **MUST** publish a birth certificate over MQTT. The birth certificate identifies the component and publishes the URL of the HTTP endpoint (`Service_URL`). Each birth certificate can register only one component function with the Directory component, regardless of whether the component is a Sparkplug Node or Device. If a single software process intends to provide multiple components, it can publish multiple Device birth certificates to achieve this.
```

Comment:
```suggestion
* A component **MUST** publish a birth certificate over MQTT. The birth certificate identifies the component and publishes the URL of the HTTP endpoint (`Service_URL`). Each birth certificate can register only one component function with the [Directory](/docs/framework-components/core-components/directory) component, regardless of whether the component is a Sparkplug Node or Device. If a single software process intends to provide multiple components, it can publish multiple Device birth certificates to achieve this.
```